dp/cloud/python/magma/configuration_controller/response_processor/response_db_processor.py

```diff
@@ -136,30 +145,3 @@ def _process_request(self, request: DBRequest) -> None:
             f"[{self.response_type}] Marking request {request} as processed.",
         )
         request.state = self.request_states_map[RequestStates.PROCESSED.value]
-
-    def _log_response(self, session: Session, response: DBResponse):
```

Comment:
Keep it, change arguments and put the `try/except` with `fluentd_client` inside, here.

Reply:
done